hadoop-mapreduce-project/hadoop-mapreduce-client/hadoop-mapreduce-client-jobclient/src/test/java/org/apache/hadoop/mapred/TestTextOutputFormat.java

```diff
@@ -165,68 +165,68 @@ public void testFormatWithCustomSeparator() throws Exception {
    */
   @Test
   public void testCompress() throws IOException {
-   JobConf job = new JobConf();
```

Comment:
this test I believe is changed to fix some checkstyles, maybe not required

Reply:
I've committed a change that reverts everything in this file since it is just a test class.